build_library/vm_image_util.sh

```diff
@@ -444,6 +445,25 @@ install_oem_package() {
     sudo rm -rf "${oem_tmp}"
 }
 
+# Write the OEM ACI file into the OEM partition.
+install_oem_aci() {
+    local oem_aci=$(_get_vm_opt OEM_ACI)
+
+    [ -n "${oem_aci}" ] || return 0
+
+    local save_BUILD_DIR=$BUILD_DIR
+    BUILD_DIR=$FLAGS_to
```

Comment:
I'm not sure how calling it as a different script would make a differnece.  The functions called in `create_prod_image` rely on `BUILD_DIR` as a result of normally being used by the `build_image` script, so that still needs to be defined.  It doesn't seem to be defined properly by the `image_to_vm` script.